podium-uaa/src/main/java/org/bbmri/podium/web/rest/OrganisationResource.java

```diff
@@ -56,6 +56,8 @@ public OrganisationResource(OrganisationService organisationService) {
     private void copyProperties(Organisation source, Organisation target) {
         target.setName(source.getName());
         target.setShortName(source.getShortName());
+        target.setActivated(source.isActivated());
+        target.setDeleted(source.isDeleted());
```

Comment:
This should not be in copyProperties. This should be in the logic in the controller method.
The idea of copyProperties is to update fields that do not change behaviour, but just change the contents.
If we do this consistently, you can reuse the copyProperties functions in different endpoints or for different user types.